.travis.yml

```diff
@@ -22,7 +22,9 @@ matrix:
 
 before_script:
   - rustup component add clippy-preview
+  - rustup component add rustfmt-preview
 script:
+  - cargo fmt --all -- --check
```

Comment:
should have that in `./build`, too once PR #4 gets merged

Reply:
Yes, I will rebase and add this here after #4 is merged